tests/sdk/v2/driver/test_client.py

```diff
@@ -1799,177 +1800,58 @@ def test_logs_decode(
-                    "wf-run-92ae97be-6f19-43dc-9c21-29153cb55f81 UPDATED Running\n  Pod Conditions:\n  -- Initialized True\n  -- Ready False (ContainersNotReady): containers with unready status: [workflow-run] \n  -- ContainersReady False (ContainersNotReady): containers with unready status: [workflow-run] \n  -- PodScheduled True\n  Container Information:\n  -- fluent-bit Running\n  -- linkerd-proxy Running\n  -- workflow-run Terminated: Exit code: 0 ",  # noqa: E501
-                    "wf-run-92ae97be-6f19-43dc-9c21-29153cb55f81 ADDED Running\n  Pod Conditions:\n  -- Initialized True\n  -- Ready False (ContainersNotReady): containers with unready status: [workflow-run] \n  -- ContainersReady False (ContainersNotReady): containers with unready status: [workflow-run] \n  -- PodScheduled True\n  Container Information:\n  -- fluent-bit Running\n  -- linkerd-proxy Running\n  -- workflow-run Terminated: Exit code: 0 ",  # noqa: E501
-                    "wf-run-92ae97be-6f19-43dc-9c21-29153cb55f81 UPDATED Running\n  Pod Conditions:\n  -- Initialized True\n  -- Ready False (ContainersNotReady): containers with unready status: [workflow-run] \n  -- ContainersReady False (ContainersNotReady): containers with unready status: [workflow-run] \n  -- PodScheduled True\n  Container Information:\n  -- fluent-bit Running\n  -- linkerd-proxy Running\n  -- workflow-run Terminated: Exit code: 0 ",  # noqa: E501
-                    "wf-run-92ae97be-6f19-43dc-9c21-29153cb55f81 UPDATED Succeeded\n  Pod Conditions:\n  -- Initialized True (PodCompleted)\n  -- Ready False (PodCompleted)\n  -- ContainersReady False (PodCompleted)\n  -- PodScheduled True\n  Container Information:\n  -- fluent-bit Terminated: Exit code: 0 \n  -- linkerd-proxy Terminated: Exit code: 0 \n  -- workflow-run Terminated: Exit code: 0 ",  # noqa: E501
-                    "wf-run-92ae97be-6f19-43dc-9c21-29153cb55f81 UPDATED Succeeded\n  Pod Conditions:\n  -- Initialized True (PodCompleted)\n  -- Ready False (PodCompleted)\n  -- ContainersReady False (PodCompleted)\n  -- PodScheduled True\n  Container Information:\n  -- fluent-bit Terminated: Exit code: 0 \n  -- linkerd-proxy Terminated: Exit code: 0 \n  -- workflow-run Terminated: Exit code: 0 ",  # noqa: E501
-                    "wf-run-92ae97be-6f19-43dc-9c21-29153cb55f81 DELETED Succeeded\n  Pod Conditions:\n  -- Initialized True (PodCompleted)\n  -- Ready False (PodCompleted)\n  -- ContainersReady False (PodCompleted)\n  -- PodScheduled True\n  Container Information:\n  -- fluent-bit Terminated: Exit code: 0 \n  -- linkerd-proxy Terminated: Exit code: 0 \n  -- workflow-run Terminated: Exit code: 0 ",  # noqa: E501
-                ]
+                sys_logs = client._get_system_logs(workflow_run_id)
+
+                assert len(sys_logs)
```

Comment:
I think this is better, but maybe we can improve it.
Have a look at Alex's tests, by checking the number of messages, the unique files, etc. https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/191/files
I wouldn't want to spend too much longer on it, we can add a ticket to improve later.